dss/dss/upload_to_cloud.py

```diff
@@ -103,10 +103,10 @@ def upload_to_cloud(file_handles, staging_bucket, replica, from_cloud=False, log
                 )
                 sums = fh.get_checksums()
                 metadata = {
-                    "hca-dss-s3_etag": sums["s3_etag"],
-                    "hca-dss-sha1": sums["sha1"],
-                    "hca-dss-sha256": sums["sha256"],
-                    "hca-dss-crc32c": sums["crc32c"],
+                    "dss-cli-s3_etag": sums["s3_etag"],
```

Comment:
this might break on the DSS end, have to verify hardcoded metadata in files API.